### DISCUSSION TITLE
CefSettings: RemoteDebuggingPort and/or CefCommandLineArgs stopped working in 126

### DISCUSSION THREAD

**Discussion by SiMoStro** [cefsharp/CefSharp#4895] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

126.2.160

### Operating System

Windows 11

### Architecture

x64

### .Net Version

4.7.2

### Implementation

WinForms

### Reproduction Steps

We've successfully used the Chromium devtools/debugging feature in all latest versions, initializing CefSettings in this way:
```
CefSettings settings = new CefSettings
{
    LogSeverity = LogSeverity.Default,
};
settings.RemoteDebuggingPort = 3333;
settings.CefCommandLineArgs.Add("--remote-allow-origins", "*");
```
starting from version 126.2.70 this has stopped working.
We are currently testing 126.2.180 but the problem persists.

### Expected behavior

The Chromium debug console should be available at the configured port:

![image](https://github.com/user-attachments/assets/23e925f1-06e4-48cf-afc7-47736c7485eb)

### Actual behavior

The page at the configured port does not generate errors but shows nothing:

![image](https://github.com/user-attachments/assets/27636e07-1737-4c23-9f2f-f5a48138fca4)

### Regression?

Yes, this used to work in previus versions.
Note that this problem has been already notified in this thread:

https://github.com/chromiumembedded/cef/issues/3740

but has been closed as "WontFix", but this is clearly a regression.

### Known Workarounds

No workaround

### Does this problem also occur in the CEF Sample Application

Not Tested

### Other information

Other users have already notified the problem:

https://github.com/chromiumembedded/cef/issues/3740

**Comment by SiMoStro** [comment]
Hi guys,

I don't get why this is now a "discussion" - anyway, attached the solutions which reproduce the bug: 

- one links CefSharp 125.0.210 (and CEF 125.0.21)
- the second links CefSharp 126.2.180 (and CEF 126.2.18)

the second one doesn't show the DevTools page.

[CefTest_125.zip](https://github.com/user-attachments/files/16492384/CefTest_125.zip)

[CefTest_126.zip](https://github.com/user-attachments/files/16492386/CefTest_126.zip)

**v125**
![test_125](https://github.com/user-attachments/assets/b75c9f4e-f352-4020-91d7-a90bd4affee1)

**v126**
![test_126](https://github.com/user-attachments/assets/6ba0069e-f3b5-40d1-ade6-1c9de381ea59)

**Reply by amaitland** [reply]
> I don't get why this is now a "discussion" - anyway,

This is an `upstream` issue and needs to be fixed in `CEF`. You are welcome to discussion possible workarounds here. It's not a bug in `CefSharp`.

The short term workaround is revert to using the alloy bootstrap. 

https://github.com/cefsharp/CefSharp/issues/4835#issue-2350564824

> Note that this problem has been already notified in this thread:
> 
> [chromiumembedded/cef#3740](https://github.com/chromiumembedded/cef/issues/3740)
> 
> but has been closed as "WontFix", but this is clearly a regression.

This isn't something that can be fixed in `CefSharp`. If the `CEF` maintainer has marked as `WONTFIX` then it will need someone else to debug and submit a `PR`.

> Does this problem also occur in the CEF Sample Application
> Not Tested

The problem reproduces with the `CEF Sample Application`. In future please perform this testing.

**Comment by amaitland** [comment]
```
# Works
.\cefclient.exe --use-alloy-style --remote-debugging-port=9955 --disable-request-handling-for-testing --disable-chrome-runtime --remote-allow-origins=http://localhost:9955

# Doesn't work
.\cefclient.exe --use-alloy-style --remote-debugging-port=9955 --disable-request-handling-for-testing  --remote-allow-origins=http://localhost:9955
```
